data/helpers.d/filesystem

```diff
@@ -80,3 +80,31 @@ properly with chmod/chown." >&2
     chmod 755 $TMP_DIR
     echo $TMP_DIR
 }
+
+# Remove a file or a directory securely
+#
+# usage: ynh_secure_remove path_to_remove
+# | arg: path_to_remove - File or directory to remove
+ynh_secure_remove () {
+	path_to_remove=$1
```

Comment:
Following the discussion we had a few weeks ago in the meeting, I think we should add a check here to refuse to do anything if we have a second argument